### PR TITLE
Added border color property to button styles

### LINKED ISSUE #4297: Add border color property to button styles
### Is your proposal related to a problem?

There's no option to set a border color to the button component. 

### Describe the solution you'd like

Add a border color property to the button component.

Screenshot 2022-10-04 at 4 38 25 PM

### PR DESCRIPTION
**Pull request by vedant-z**
Fixes #4297

### CODE REVIEW

**Comment by vedant-z** [comment]
@withshubh Have I done the required changes correctly?

**Comment by withshubh** [comment]
> @withshubh Have I done the required changes correctly?

Let's wait for the PR to be reviewed

File: frontend/src/Editor/WidgetManager/widgetConfig.js [hunk 1]
```diff
@@ -499,6 +507,7 @@ export const widgets = [
         loaderColor: { value: '' },
         visibility: { value: '{{true}}' },
         borderRadius: { value: '{{0}}' },
+        borderColor: { value: '' },
```
Comment (arpitnath) [inline frontend/src/Editor/WidgetManager/widgetConfig.js:510]: Can we add a default value for the border color?
```suggestion
        borderColor: { value: '#375FCF' },
```

Comment (arpitnath) [reply frontend/src/Editor/WidgetManager/widgetConfig.js:510]: other than that LGTM!
@vedant-z 🚀

**Review by arpitnath — CHANGES_REQUESTED** [review]

**Comment by vedant-z** [comment]
@arpitnath Done ✅

**Review by arpitnath — APPROVED** [review]

**Review by kavinvenkatachalam — APPROVED** [review]

**Comment by withshubh** [comment]
Hey @vedant-z 
Thanks for contributing to ToolJet. :raised_hands: 
Please submit the form to claim your Hacktoberfest swag: https://airtable.com/shr1qR6xhqsWO7LD7

**Comment by vedant-z** [comment]
Thanks so much @withshubh 

I love contributing to ToolJet and will continue contributing 🔥🔥

**Pull request merged**